.github/workflows/run-tests.yml

```diff
@@ -34,5 +34,5 @@ jobs:
                   composer update --prefer-stable --prefer-dist --no-interaction --no-suggest
 
             - name: Execute tests
-              run: vendor/bin/phpunit --testdox
```

Comment:
@tylerwoonton I changed this to get the tests to output the actual error, I can revert though if you prefer?